test/stripe/account_external_accounts_operations_test.rb

```diff
@@ -48,7 +48,8 @@ class AccountExternalAccountsOperationsTest < Test::Unit::TestCase
           @external_account_id
         )
         assert_requested :delete, "#{Stripe.api_base}/v1/accounts/#{@account_id}/external_accounts/#{@external_account_id}"
-        assert external_account.is_a?(Stripe::BankAccount)
+        assert external_account.deleted
+        assert_equal @external_account_id, external_account.id
```

Comment:
I had to fix this one test as well — it was incorrectly asserting that the object being returned from a delete action was a bank account, which isn't the case (it's actually a deleted object that comes back).